tests/unittests/test_handler/test_handler_reset_rmc.py

```diff
@@ -0,0 +1,48 @@
+from cloudinit.config import cc_reset_rmc
+
+from cloudinit import util
+
+# from cloudinit.config.cc_write_files import write_files
+from cloudinit.tests import helpers as t_help
+from cloudinit.tests.helpers import mock
+
+import logging
+import os
+
+LOG = logging.getLogger(__name__)
+NODE_FILE = '/etc/ct_node_id'
+RSCT_PATH = '/opt/rsct/install/bin'
+
+
+def recfgct():
+    content = '34fhs7kl892840f45'
+    util.write_file(NODE_FILE, content, mode=0o777, omode='w')
+    return util.load_file(NODE_FILE)
+
+
+def rmcctrl():
+    return ('', '')
+
+
+class TestRsctNodeFile(t_help.FilesystemMockingTestCase):
+    @mock.patch(
+        'cloudinit.config.cc_reset_rmc.reconfigure_rsct_subsystems',
+        side_effect=recfgct)
+    @mock.patch(
+        'cloudinit.config.cc_reset_rmc.rmcctrl',
+        side_effect=rmcctrl)
+    def test_rsct_node(self, reconfigure_rsct_subsystems, rmcctrl):
```

Comment:
This test is somewhat fine for a "happy path", you mock out 'reconfigure_rsct_subsystems', which you've asserted will modify the contents of NODE_FILE.  and then you check that the NODE_FILE has been updated (which it will have been unless there is a bug in your recfgct() function above.  

So really what this test should be called is 'test_reset_rmc_does_not_raise_exc_if_node_file_updated'.

More interesting is probably to test the unhappy path, that if reconfigure_rsct_subsystems does *not* update NODE_FILE, that you reset_rmc will raise an exception.

So I suggest adding a test: `test_reset_rmc_does_raises_unless_node_file_updated`.